js/lib/appUrlUtil.js

```diff
@@ -130,7 +130,7 @@ module.exports.isTargetAboutUrl = function (input) {
  */
 module.exports.isUrl = function (input) {
```

Comment:
yes I agree it was only ever added here at some point because urlUtil used to be an external module we couldn't modify.